cli/README.md

```diff
@@ -224,6 +224,108 @@ model User {
 
 During introspection, the Canonical Schema Mapping is used to decide if a field should have the `@updatedAt` attribute applied. It dictates that this should never happen.
 
+### Introspection Configuration
+
+The Canonical Schema Mapping is designed to ensure that the introspection result is always easy to understand. It is a design goal to not perform any magic. As a result, often it is required to manually extend the schema after introspecting a database. This is tedious and leads to extra work when re-introspecting, because manual edits will be lost.
+
+Introspection Configuration allows the user to change the Canonical Schema Mapping to be in line with patterns used in the database. 
+
+The following configuration options are awailable:
+
+- **fieldPatterns**: Add attributes to a field by matching on its name
+- **modelPatterns**: Add attributes to a model by matching on its name
+
+The configuration is stored in a `prismarc.json` file next to the `schema.prisma` file. The following example will
```

Comment:
Why this filename? Wouldn't a `introspection-config.json` be clearer in the `prisma` folder?

Reply:
Agree `rc` ending does not evoke anything to me.